spec/controllers/items_controller_spec.rb

```diff
@@ -81,10 +76,11 @@
       assigns[:error].should == 'no_accounts'
     end
 
-    context "restricted item" do
+    context "when the item requires approval" do
       before :each do
-        @item.update_attributes(:requires_approval => true)
+        @item.update_attributes(requires_approval: true)
       end
+
       it "should show a notice if you're not approved" do
```

Comment:
Good catch. Yet another reason to refactor/consolidate the various product controller specs.